langstream-pulsar/src/main/java/ai/langstream/pulsar/PulsarClusterRuntimeConfiguration.java

```diff
@@ -15,20 +15,26 @@
  */
 package ai.langstream.pulsar;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Map;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
 
-@Data
-@NoArgsConstructor
-@AllArgsConstructor
-public class PulsarClusterRuntimeConfiguration {
+public record PulsarClusterRuntimeConfiguration(
+        Map<String, Object> admin,
+        Map<String, Object> service,
+        Map<String, Object> authentication,
+        @JsonProperty("default-tenant") String defaultTenant,
```

Comment:
this is clearly a breaking change, are we ok with that ? @eolivelli

Reply:
We talked about this with @eolivelli  .It's OK. Nobody could use Pulsar before.

Reply:
I agree